exchange.lua

```diff
@@ -1018,17 +1016,17 @@ end
 
 
 -- On success, returns true and a list of inbox entries.
--- On failure, returns false and an error message.
+-- On failure, an error message.
```

Comment:
The only place where the function returns sth is the end and there it returns true, l think:
```return true, res```
As far as l know if there's no ```return false, msg```, the function never returns it, and l thought an error message could mean that the error() function is called somewhere.